main.go

```diff
@@ -188,6 +189,26 @@ func main() {
 		authorizers = append(authorizers, externalAuthorizer)
 	}
 
+	var jwtFromExtraProviderAuthenticator authenticators.Authenticator
+	// Add the jwt extra authentication
+	if c.JWTFromExtraProviderEnabled {
+		if !slices.Contains([]string{"http", "https"}, c.JWTFromExtraProviderProviderURL.Scheme) {
```

Comment:
nit: i would usually put these kind of checks in the New function. not sure if that's done all around the code though; so :shrug:

Reply:
Yes, let me move them.